EpsilonLights/Core/Src/main.c

```diff
@@ -321,32 +334,31 @@ static void MX_CAN_UserInit(void)
     batteryFilterConfig.FilterMaskIdLow = 0; //unused
     batteryFilterConfig.FilterFIFOAssignment = 0;
     batteryFilterConfig.FilterActivation = ENABLE;
-    batteryFilterConfig.BankNumber = 0; // Set all filter banks for CAN2
+    batteryFilterConfig.SlaveStartFilterBank = 0; // Set all filter banks for CAN2
```

Comment:
I'm not sure what `SlaveStartFilterBank` does. Is the comment still correct?

Reply:
I think what it does is that it assigns all of the lower index filter banks to the master CAN instance and the higher ones to the slave CAN instance. It's only used for dual CAN instances.

Reply:
When I was looking at the new API, I'm pretty sure they just refactored `BankNumber` to be `SlaveStartFilterBank`